README.md

```
@@ -1 +1,75 @@
+
+for creating partitions:
+brew install pg_partman
+
+
+------------new --------------------
+created a stock_analysis schema
+grant usage and permissoon on the stock_analysis schema to stockanalysisservice user
+
+GRANT CREATE ON DATABASE stock_analysis TO stockanalysisservice;
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)